website/pages/en/index.js

```diff
@@ -131,66 +126,43 @@ class Index extends React.Component {
 								<div className='col-12 col-md-4 mb-4'>
 									<div className='d-flex align-items-center mb-2'>
 										<img className='mr-2' src='/img/glyphs/rectangle-1.svg' width='40' />
-										<h3 className='mb-0'>Fast and efficient database</h3>
+										<h3 className='mb-0'>Fast and Efficient Database</h3>
 									</div>
 								</div>
 								<div className='col-12 col-md-4 mb-4'>
 									<div className='d-flex align-items-center mb-2'>
 										<img className='mr-2' src='/img/glyphs/rectangle-2.svg' width='40' />
-										<h3 className='mb-0'>Modular P2P networking stack in libp2p</h3>
+										<h3 className='mb-0'>Modular P2P Networking Stack</h3>
 									</div>
 								</div>
 								<div className='col-12 col-md-4'>
 									<div className='d-flex align-items-center mb-2'>
 										<img className='mr-2' src='/img/glyphs/rectangle-3.svg' width='40' />
-										<h3 className='mb-0'>Hot-swappable consensus layer</h3>
+										<h3 className='mb-0'>Hot-Swappable Consensus Layer</h3>
 									</div>
 								</div>
 								<div className='col-12 col-md-4 mb-4'>
 									<div className='d-flex align-items-center mb-2'>
 										<img className='mr-2' src='/img/glyphs/rectangle-4.svg' width='40' />
-										<h3 className='mb-0'>Customizable transaction queue management system</h3>
+										<h3 className='mb-0'>Flexible Transaction Queue</h3>
 									</div>
 								</div>
 								<div className='col-12 col-md-4 mb-4'>
 									<div className='d-flex align-items-center mb-2'>
 										<img className='mr-2' src='/img/glyphs/rectangle-5.svg' width='40' />
-										<h3 className='mb-0'>Rich framework for composing runtime logic.</h3>
+										<h3 className='mb-0'>Rich Runtime Library</h3>
```

Comment:
I know you want to make this simpler, but lots of SDKs give you a library. Substrate's edge is the composition framework.
```suggestion
										<h3 className='mb-0'>Rich Runtime Library and Flexible Composition</h3>
```

Reply:
Same as above...needed to keep this super terse so amended your suggestion slightly.